groupbalancer.go

```diff
@@ -2,33 +2,41 @@ package kafka
 
 import "sort"
 
-// strategy encapsulates the client side rebalancing logic
-type strategy interface {
-	// ProtocolName of strategy
+// GroupMember describes a single participant in a consumer group.
+type GroupMember struct {
+	// ID is the unique ID for this member as taken from the JoinGroup response.
+	ID       string
```

Comment:
Are you running an old-school version of `go fmt`? Or maybe this is a new formatting rule? The spacing here is unusual.

Reply:
My on-save action got unhooked, so `go fmt` wasn't running.  Good catch!